### PR TITLE
The template to run maestro in ROSA

### PR DESCRIPTION
**Pull request by skeeey**

### CODE REVIEW

File: templates/service-template-rosa.yml [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+    apiVersion: apps/v1
+    metadata:
+      name: maestro
+      labels:
+        app: maestro
+    spec:
+      selector:
+        matchLabels:
+          app: maestro
+      replicas: ${{SERVER_REPLICAS}}
```
Comment (clyang82) [inline templates/service-template-rosa.yml:162]: no error for 2 braces?

Comment (skeeey) [reply templates/service-template-rosa.yml]: no errors, it works, and I will use 1 brace to keep same code style with others

**Review by clyang82 — APPROVED** [review]
LGTM

**Pull request merged**